### PR TITLE
aws ccm

### PR DESCRIPTION
**Pull request by huali9**
add the workflow "cucushift-installer-rehearse-aws-ipi-ccm" and provision & deprovision chains

@jhou1 @sunzhaohua2 @miyadav Please help to take a look, thanks!

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ccm/cucushift-installer-rehearse-aws-ipi-ccm-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-ccm
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-ccm-provision
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-ccm-deprovision
+  documentation: |-
+    This is the workflow to trigger Prow's rehearsal test when submitting installer steps/chain/workflow
```
Comment (sunzhaohua2) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ccm/cucushift-installer-rehearse-aws-ipi-ccm-workflow.yaml:9]: we'd better add the supported versions in workflow documentation  as well.

**Comment by jhou1** [comment]
Overall looks good to me. Please add the supported version in the document, and then add it to the installer-rehearsal job(`ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml`) to trigger a test for this. 

/assign @jianlinliu

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ccm/cucushift-installer-rehearse-aws-ipi-ccm-workflow.yaml (hunk 1 above)
Comment (huali9) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ccm/cucushift-installer-rehearse-aws-ipi-ccm-workflow.yaml]: > we'd better add the supported versions in workflow documentation as well.

added, thanks

**Comment by jianlinliu** [comment]
1. Validation job failed, pls check
2. Pls always squash multiple commits into one.

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml [hunk 2]
```diff
@@ -47,6 +47,13 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+- as: installer-rehearse-aws-ccm-qe
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml:50]: No need to add this new test item, just replace the above line 49 with the your workflow for the rehearsed testing.

**Comment by jianlinliu** [comment]
1. Validation job failed, pls always check the job carefully (if https://github.com/openshift/release/pull/29205#discussion_r891115480 is resolved, maybe no such failure.).
2. Pls always squash multiple commits into one.

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml [hunk 3]
```diff
@@ -46,7 +46,7 @@ tests:
     cluster_profile: aws-usgov-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+    workflow: cucushift-installer-rehearse-aws-ipi-ccm
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml:49]: Sorry, maybe I point you to the wrong place, the change should happen on the line 42

**Comment by jhou1** [comment]
The step `ccm-conf-apply-feature-gate` has failed. https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/29205/rehearse-29205-periodic-ci-openshift-verification-tests-master-installer-rehearse-installer-rehearse-aws-qe/1534356958662889472/artifacts/installer-rehearse-aws-qe/ccm-conf-apply-feature-gate/build-log.txt
/retest

**Comment by huali9** [comment]
It takes nearly 30 minutes to enable CCM on 4.10, but only nearly 20 minutes to enable CCM on 4.11, so turns timeout on 4.10. After https://github.com/openshift/release/pull/29360 merged, "Timed out waiting for result of CCMPodsCreated" error should be solved, let's wait for the pr merged then check the result.

**Comment by huali9** [comment]
65/65 Tests Passed! @sunzhaohua2 @jhou1 @miyadav @jianlinliu PTAL, thanks!
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29205/rehearse-29205-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.10-installer-rehearse-aws-qe/1536522680998563840

**Comment by huali9** [comment]
just rebase the code,  @sunzhaohua2 @jhou1 @miyadav @jianlinliu PTAL again, thanks!

**Pull request merged**